packages/amplify-cli-core/src/__tests__/amplify-lockfile-dependency-detector/amplify-nodejs-detector.test.ts

```diff
@@ -26,7 +27,7 @@ const npmPackageManager = getPackageManagerByType('npm');
 
 describe('parsing yarn lock files', () => {
   it('throws error when lock file not found', async () => {
-    yarnPackageManager.lockFile = 'yarn-test-not-found.lock';
+    (yarnPackageManager as $TSAny).lockFile = 'yarn-test-not-found.lock';
```

Comment:
Seems like there's probably a way to updates these tests to check the same behavior without overriding this value, but I'm okay if that is out of scope for now